specs/trust-store-trust-policy.md

```diff
@@ -427,11 +425,12 @@ To check the revocation status of a certificate using OCSP, the following steps
    This step also includes verifying the OSCP response signing certificate is valid and trusted.
    The OCSP signing certificate must be issued by the same CA as the certificate being verified or the OCSP response must be signed by the issuing CA.
 1. Verify that the OCSP response is valid (not expired).
-   A CRL is considered expired if the current date is after the `NextUpdate` field in the CRL.
+   Response is considered expired if the current date is after the `NextUpdate` field in the response.
 1. Verify that the OCSP response indicates that the certificate is not revoked i.e `CertStatus` is `good`.
     1. If the certificate is revoked i.e `CertStatus` is `revoked`, look for `InvalidityDate`.
        If the invalidity date is present and timestamp signature is also present then if the invalidity date is before the timestamping date, the certificate is considered revoked.
        If the invalidity date is not present in OCSP response, the certificate is considered revoked.
+    1. If the certificate status is `unknown`, the revocation status is considered `revocation unavailable`.
 1. If `id-pkix-ocsp-nocheck`(1.3.6.1.5.5.7.48.1.5) extension is not present on the OCSP signing certificate then revocation checking must be performed using CRLs for the OCSP signing certificate.
 
 ## FAQ
```

Comment:
Maybe we can remove the following "TODO"?

**Q: Does Notary v2 support overriding of revocation endpoints to support signature verification in disconnected environments?**
**A:** TODO: Update after verification extensibility spec is ready.

Reply:
Yes will do it in separate PR